validator/accounts/account.go

```diff
@@ -39,13 +40,10 @@ func VerifyAccountNotExists(directory string, password string) error {
 // generates a BLS private and public key, and then logs the serialized deposit input hex string
 // to be used in an ETH1.0 transaction by the validator.
 func NewValidatorAccount(directory string, password string) error {
-	// First, if the keystore already exists, throws an error as there can only be
-	// one keystore per validator client.
-	if err := VerifyAccountNotExists(directory, password); err != nil {
-		return fmt.Errorf("validator account exists: %v", err)
-	}
-	shardWithdrawalKeyFile := directory + params.BeaconConfig().WithdrawalPrivkeyFileName
-	validatorKeyFile := directory + params.BeaconConfig().ValidatorPrivkeyFileName
+	t := time.Now()
+
+	shardWithdrawalKeyFile := directory + params.BeaconConfig().WithdrawalPrivkeyFileName + t.Format("20060102150405")
```

Comment:
as @nisdas pointed out, is there a better way to mark them? maybe use the pubkey for suffix ? 
Having the time doesnt really help in differentiating them...

Reply:
Pubkey substring sounds good to me. Maybe take the first 12 characters of the key. 

The full key is way too long

Reply:
I merged this pr into #2069 and resolved it there.